.github/workflows/ci.yml

```diff
@@ -217,8 +217,63 @@ jobs:
       - run: bun install
       - name: Performance measurement of type check
         run: |
-            bun scripts/generate-app.ts
-            bun tsc -p tsconfig.build.json --diagnostics | bun scripts/process-results.ts > diagnostics.json
+          bun scripts/generate-app.ts
+          bun tsc -p tsconfig.build.json --diagnostics | bun scripts/process-results.ts > diagnostics.json
+        working-directory: perf-measures/type-check
+      - name: Performance measurement of bundle check
+        run: |
+          bun run build
+          bun perf-measures/bundle-check/scripts/check-bundle-size.ts > perf-measures/bundle-check/size.json
+      - name: Run octocov (main)
+        uses: k1LoW/octocov-action@v1
+        with:
+          config: perf-measures/.octocov.perf-measures.main.yml
+        env:
+          OCTOCOV_GITHUB_REF: refs/heads/main
+          OCTOCOV_CUSTOM_METRICS_BENCHMARK_BUNDLE: perf-measures/bundle-check/size.json
+          OCTOCOV_CUSTOM_METRICS_BENCHMARK_TYPE: perf-measures/type-check/diagnostics.json
+
+  perf-measures-tsgo-type-and-bundle-check-on-pr:
```

Comment:
There are the same steps like a `Performance measurement of type check` in the following actions:

* `perf-measures-type-and-bundle-check-on-p`
* `perf-measures-type-and-bundle-check-on-main`
* `perf-measures-tsgo-type-and-bundle-check-on-pr`
* `perf-measures-tsgo-type-and-bundle-check-on-main`

To reduce the code, how about using [composite action](https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action)?

Reply:
Once it becomes TypeScript 7.0, the number of jobs will decrease to two, so I wouldn't need to make them common/shared.
However, a good idea came to mind when I thought about composite actions, so I'll make one.